### PR TITLE
Add Telegram and WhatsApp

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by mxxcon** [comment]
> Another addition to the Privacy and Security section is Two Step Verification. It allows you to set up a password that will be required every time you log into your account from a new device – in addition to the code you get in the SMS. 

So telegram uses just another static password beside login password and sms? That's kinda a weird implementation. :confused:

**Comment by Eitot** [comment]
There is no login password, just SMS verification. You could easily login on another device if you get access to someone’s phone. The 2FA now requires an additional password that can be reset by e-mail recovery. Not sure whether that is really all that more secure (since smartphones typically receive e-mails too) but it’s 2FA regardless.

**Comment by mxxcon** [comment]
Anybody else take a look at this?

**Comment by Carlgo11** [comment]
a more compressed version of the fruux logo
fruux logo

File: _data/communication.yml [hunk 1]
```diff
@@ -97,12 +97,27 @@ websites:
...
+      doc: https://telegram.org/blog/sessions-and-2-step-verification
+
     - name: ToutApp
       url: http://www1.toutapp.com/
       twitter: toutapp
       img: toutapp.png
       tfa: No
 
+    - name: WhatsApp
+      url: http://www.whatsapp.com/
```
Comment (Carlgo11) [inline _data/communication.yml]: this link should be https://www.whatsapp.com/

**Comment by jamcat22** [comment]
I think we should add Telegram. You can disable email recovery for the password if you want. SMS/Phone Call is always required to login, the password just adds the knowledge factor.

**Comment by jamcat22** [comment]
I say change it to `sms: Yes` and `phone: Yes` and remove the email and software tags.

**Comment by mxxcon** [comment]
@jamcat22 Why `phone: yes`? Telegram doesn't make phone calls to verify

**Comment by jamcat22** [comment]
Here are some photos:

Signing in:
image
image

Resetting with email:
image

Resetting with no email:
image

Setup:
image

Optional hint:
image

Skipping recovery email entry:
image

Settings page:
image

**Comment by jamcat22** [comment]
@mxxcon Yes they do.
image

**Comment by mxxcon** [comment]
:+1:

**Comment by ghost** [comment]
Any updates on this? :-)

**Comment by mxxcon** [comment]
@Eitot Carl asked you to update whatsup url to https...

**Comment by ghost** [comment]
@mxxcon Ah… For some reason I thought these were already added to the PR.

**Comment by ghost** [comment]
I added the proposals by @Carlgo11 and @jamcat22 now. Sorry about that!

**Comment by jamcat22** [comment]
Do we think it `looks-good` @mxxcon

**Comment by mxxcon** [comment]
@Carlgo11 why did it touch Fluux image in this PR? :confused:

**Comment by mxxcon** [comment]
There's now a merge conflict...I think it's because of fruux image?

**Comment by ghost** [comment]
I made the PR for fruux originally, but I didn't compress the image. I thought I'd fix that in one go.

**Comment by Carlgo11** [comment]
Remove that commit then since the PR has nothing to do with fruux?

**Comment by ghost** [comment]
@Carlgo11 Done.

**Comment by mxxcon** [comment]
hrm, it still shows that this PR can't be cleanly merged...

**Comment by Carlgo11** [comment]
Probably because he did a revert commit instead of just removing the initial commit

**Comment by ghost** [comment]
How can I remove commits?

**Comment by mxxcon** [comment]
@Eitot perhaps scrap this PR, make sure that your forked repo is up to date with this one and create a new PR? That will ensure only clean and necessary commits go in. :wink:

**Comment by Carlgo11** [comment]
@Eitot if you add me as a collaborator on https://github.com/Eitot/twofactorauth I can push a working version to this PR.

**Comment by ghost** [comment]
Done.

**Comment by mxxcon** [comment]
Why are you deleting SparkPost? :/

**Comment by Carlgo11** [comment]
There. That should do it.

File: _data/communication.yml [hunk 2]
```diff
@@ -104,10 +104,25 @@ websites:
...
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      doc: https://telegram.org/blog/sessions-and-2-step-verification
+
     - name: ToutApp
       url: http://www1.toutapp.com/
       twitter: toutapp
       img: toutapp.png
+      tfa:
```
Comment (mxxcon) [inline _data/communication.yml]: This line doesn't seem correct..

Comment (Carlgo11) [inline _data/communication.yml]: whops, fixed now. :see_no_evil:

**Pull request merged**